src/utils/httpUtils.ts

```diff
@@ -11,3 +13,7 @@ export function getURL(baseURL: string, path: string, orgId?: string): string {
 function routeToGateway(baseURL: string): boolean {
   return baseURL.includes('snykgov.io');
 }
+
+function isValidOrg(orgId?: string): boolean {
```

Comment:
enh: `orgId: string | undefined` - there isn't much of a reason of supporting `isValidOrg()`

Reply:
We are validating the existence of the orgId, in a FedRamp environment. We also want to validate the orgId format, in case someone uses code client, and sends the org name instead, to not send a request all the way to the api-gateway, because it will fail anyway.

Reply:
Was this meant as a reply to this? :)

>What's the reasoning behind this change? I can't quite determine the context from the information in the ticket and the PR?

What I'm trying to understand is why the client-side validation here is necessary - is it just for better error messaging?

Reply:
Indeed, a better error handling is the main reason. If an incorrect org id reaches the api-gateway, the error is not very clear, and hard to debug the reason of failure.